src/index.js

```diff
@@ -0,0 +1,132 @@
+import './style.css';
+import Draggables from './drag';
+import statusCheck from './status';
+import uiUpdate from './updates';
+
+class TaskList {
+  refereceList = [];
+
+  container = document.querySelector('.container');
+
+  addTaskBtn = document.querySelector('.add-btn');
+
+  listDraggables = new Draggables(this.container);
+
+  constructor(taskList) {
+    this.taskList = taskList;
+
+    this.container.addEventListener('drop', () => {
+      this.save();
+    });
+
+    this.addTaskBtn.addEventListener('click', () => {
+      this.createTodoItem({
+        description: document.querySelector('.add-list').value,
+        completed: false,
+      });
+      this.listDraggables.updateList();
+      this.save();
+      document.querySelector('.add-list').value = '';
+    });
+  }
+
+  save = () => {
+    const container = document.querySelector('.container');
+    const allElements = container.getElementsByTagName('li');
+
+    this.clearTaskList();
```

Comment:
I think that when creating classes if a method calls another method is best to define this method before. It won't affect your code, but it will make it easier to read.